cinema_service/settings.py

```
@@ -86,8 +87,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ["POSTGRES_DB"],
+        "USER": os.environ["POSTGRES_USER"],
+        "PASSWORD": os.environ["POSTGRES_PASSWORD"],
```

Same recommendation for POSTGRES_PASSWORD: avoid direct os.environ[...] access. If you intentionally want the app to fail loudly when password is missing, you could still add an explanatory error message to help debugging.